plugins/parcelize/parcelize-compiler/parcelize.backend/src/org/jetbrains/kotlin/parcelize/AndroidSymbols.kt

```diff
@@ -185,6 +186,12 @@ class AndroidSymbols(
         }
     }.symbol
 
+    private val kotlinDuration: IrClassSymbol = createClass(
```

Comment:
Maybe call this `kotlinTimeDuration` to follow other naming such as `androidOsParcel`?

Reply:
Done